### ISSUE TITLE
[Accessibility]: Need better description for lines added / removed / modified and better description for author

### ISSUE THREAD

**Issue by jasongornall** [sourcegraph/sourcegraph#36252]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34431 

### Problem description

![image](https://user-images.githubusercontent.com/6912393/171050780-3434f736-f6f8-4d8e-918b-eca659b88455.png)
Needs improved descriptions for labels so that the context is relevant. Currently only surfacing line numbers and name.

### Expected behavior
https://sourcegraph.com/github.com/sourcegraph/sourcegraph-code-discussions/-/commit/37873123e6fdf7706e6404f66d6a151e7d9a0cfa?visible=1#diff-b0678b51723ef8450026fa0f6967216fR22
Line 17 Added
Line 18 Modified
Line 19 Removed

### Additional details

This may not be possible and could be a limitation of the editor. Unsure.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
